data/pref.xml

```diff
@@ -317,9 +317,9 @@
       <option id="discard_brush" type="bool" default="false" />
     </section>
     <section id="shared">
-      <option id="share_dynamics" type="bool" default="false" />
       <option id="share_ink" type="bool" default="false" />
       <option id="ink" type="app::tools::InkType" default="app::tools::InkType::DEFAULT" />
+      <option id="share_dynamics" type="bool" default="true" />
```

Comment:
Not sure why this line changed to this place now, but you can just change `false` to `true`.